app/commands/solution/generate_help_file.rb

```diff
@@ -31,21 +31,30 @@ def tests
     def submitting
       submitting_solution_intro_text = I18n.t("exercises.documents.submitting_solution_intro").strip
 
+      # TODO: (Required) refine submitting solution text
+      submitting_solution_text = I18n.t("exercises.documents.submitting_solution",
+        { solution_file_paths: solution.exercise.git.solution_filepaths.join(" ") }).strip
```

Comment:
Now that we have the solution filepaths, we can have the submit instructions contain the actual command that student should use to submit, so less chance of wrongly submitted files I hope!

Reply:
Nice!!